File: src/main/java/scrumweb/dto/fieldcontent/TextAreaContentDto.java
package scrumweb.dto.fieldcontent;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@Getter
@Setter
public class TextAreaContentDto extends FieldContentDto {

    private String content;

    public TextAreaContentDto(Long projectFieldId, String projectFieldName, String content) {
        super(projectFieldId, projectFieldName);
        this.content = content;
    }
}

File: src/frontend/src/app/model/fields-content/TextAreaContentDto.ts
import { FieldContentDto } from './FieldContentDto';

export class TextAreaContentDto extends FieldContentDto {
    content: string;

    constructor(projectFieldId: number, projectFieldName: string, content: string) {
        super(projectFieldId, projectFieldName);
        this.content = content;
    }
}
